Makefile

```diff
@@ -256,17 +256,31 @@ bundle: operator-sdk manifests kustomize kubectl ## Generate bundle manifests an
 	sed -i '/^[[:space:]]*createdAt:/d' bundle/manifests/oran-o2ims.clusterserviceversion.yaml
 
 .PHONY: bundle-build
-bundle-build: bundle ## Build the bundle image.
+bundle-build: bundle docker-push ## Build the bundle image.
 	$(CONTAINER_TOOL) build -f bundle.Dockerfile -t $(BUNDLE_IMG) .
 
 .PHONY: bundle-push
-bundle-push: ## Push the bundle image.
+bundle-push: bundle-push ## Push the bundle image.
```

Comment:
Hmm, should this actually be `bundle-push: bundle-build`?

Reply:
LGTM otherwise

Reply:
Yep, cut-n-paste error there... thanks.